static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -121,7 +197,12 @@ const Context = ({
           const showStacktraceLink = hasStacktraceLink && isActive;
 
           return (
-            <StyledContextLine key={index} line={line} isActive={isActive}>
+            <StyledContextLine
+              key={index}
+              line={line}
+              isActive={isActive}
+              color={isActive ? 'transparent' : lineColors[index]}
```

Comment:
lets just fall back to transparent and not create an array of 'transparent'
```suggestion
              color={isActive ? 'transparent' : lineColors[index] ?? 'transparent'}
```